samples/log_prob.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @license
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {GoogleGenerativeAI} from "@google/generative-ai";
+
+async function enableLogProb() {
+    // [START log probability]
+    const genAI = new GoogleGenerativeAI(process.env.API_KEY);
+    const model = genAI.getGenerativeModel(
+        {
+            // Only 002 models + flash 1.5 8b models are enabled with log probs 
+            // option.
+            model: "gemini-1.5-flash-002",
+            generationConfig: {
+                responseLogprobs: true
+            },
+        },
+    );
+    const prompt = "Hello!";
+    const result = await model.generateContent(prompt);
+    console.log(result.response);
```

Comment:
Does this display what you are expecting? Are you trying to show a certain property on the response? Or the text of the response? If you want to show text, `console.log(result.response.text()`. If you want to show a specific property, probably drill down like `console.log(result.response.candidates[0].logProbsResult)`. Otherwise I think you're going to end up with it shortening nested objects and not seeing what's in logProbsResult.